src/test/views/ArticleBody.test.js

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable react/jsx-filename-extension */
+import React from 'react';
+import { configure, shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import ArticleBody from '../../views/ArticleBody';
+import mockArticle from '../__mocks__/mockSingleArticleData';
+
+configure({ adapter: new Adapter() });
```

Comment:
#### Nitpick
- Let's have one instance of this please.

Reply:
This has been resolved